### ISSUE TITLE
menu2.php

### ISSUE THREAD

**Issue by sercan447** [sercan447/deve#10]
<!DOCTYPE html><html class=''>
<head><meta charset='UTF-8'><meta name="robots" content="noindex">
    <script src='http://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js'></script>
<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
<link rel='stylesheet' href='normalize.css'><script src='prefixfree.min.js'></script><script src='modernizr.js'></script>
<style>
  </style></head>
<body>
<div class="container">
			  <header class="clearfix">
				    <h1>Bul Beni <span>Kaybolmuşum</span></h1>	
			  </header>
			  <div class="main">
                              <p>
                                  İÇERİK
                              </p>
                              
                                    <div class="side">
					      <nav class="dr-menu">
						        <div class="dr-trigger">
          <span class="dr-icon dr-icon-menu"></span>
          <a class="dr-label">İçerik Yönetimi</a>
        </div>
						        <ul style="color:black;">
							          <li><a class="dr-icon dr-icon-user" href="#" style="color:black;">Mekan Bilgileri</a></li>
							          <li><a class="dr-icon dr-icon-camera" href="#" style="color:black;">Grafiksel Bilgiler</a></li>
							          <li><a class="dr-icon dr-icon-heart" href="#" style="color:black;">Rezervasyon Yapan</a></li>
							          <li><a class="dr-icon dr-icon-bullhorn" href="#"  style="color:black;">Anket Sistemi</a></li>
							          <li><a class="dr-icon dr-icon-download" href="#"  style="color:black;">Mesajlar</a></li>
							          <li><a class="dr-icon dr-icon-settings" href="#"  style="color:black;">Çıkış</a></li>
							        
						        </ul>
					      </nav>
				    </div>
			  </div>
		</div>
<footer>

   
    <p><b>FOOTER</b></p>
    
</footer>

<script>
    var YTMenu = function () {
    function init() {
        [].slice.call(document.querySelectorAll('.dr-menu')).forEach(function (el, i) {
            var trigger = el.querySelector('div.dr-trigger'), icon = trigger.querySelector('span.dr-icon-menu'), open = false;
            trigger.addEventListener('click', function (event) {
                if (!open) {
                    el.className += ' dr-menu-open';
                    open = true;
                }
            }, false);
            icon.addEventListener('click', function (event) {
                if (open) {
                    event.stopPropagation();
                    open = false;
                    el.className = el.className.replace(/\bdr-menu-open\b/, '');
                    return false;
                }
            }, false);
        });
    }
    init();
}();
</script>
</body></html>
